e2e/cacheClient.e2e.ts

```diff
@@ -0,0 +1,36 @@
+import { initWithPrivateKeySigner, setCacheConfig } from "../src";
+
+const rpcUrl = "https://volta-rpc.energyweb.org";
+const privateKey = "";
+
+describe.skip("Cache client", () => {
```

Comment:
Why `skip`?

Reply:
This test can't be run during CI, because cache server is not included in test setup

Reply:
We could consider using docker here with ICS image. But due to short deadline for Origin team we can postpone this and create task for later

Reply:
Yes, this is least what can be done now